.github/workflows/release.yaml

```diff
@@ -0,0 +1,54 @@
+---
+name: Github Release
+
+on:
+  push:
+    branches:
+    - main
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    if: "contains(github.event.head_commit.message, 'chore(release):')"
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - id: version
+        name: Determine release version
+        run: |
+          version=$(npx standard-version --dry-run | grep 'tagging release' | rev | cut -d ' ' -f 1 | rev | cut -c2-)
```

Comment:
what installs this package? `npx standard-version` wouldn't it wait for user input if it is not installed already?